client/components/editor/teaching-elements/Assessment/TextSelection/Selector.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div
+    @mouseup="onMouseup">
+    <div
+      v-html="structure"
+      :class="{ editing: this.isEditing }"
+      class="selector ql-editor"/>
```

Comment:
Hmm, there are few issues with this template:
1. `<div>`'s can't be self closed
2. What's the point of having additional wrapper? Why haven't you just attach this `mouseup` listener onto `.selector.ql-editor` directly?